CMakeLists.txt

```diff
@@ -87,3 +87,7 @@ endif()
 if(BUILD_EXAMPLES)
     add_subdirectory(example)
 endif()
+
+if(BUILD_BENCH_TESTS)
+    add_subdirectory(bench_test)
+endif()
```

Comment:
Check this issue
https://github.com/NilFoundation/crypto3/issues/146

I've already made it in hashes
https://github.com/NilFoundation/crypto3-hash/blob/cc48e2ead139c7c7f190b3945c14c2237948805c/CMakeLists.txt#L282